images/openshift-proxy-pull-test.yml

```diff
@@ -0,0 +1,22 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/machine-config-operator
+content:
+  source:
+    dockerfile: Dockerfile.proxy
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/machine-config-operator.git
+enabled_repos:
+- rhel-8-baseos-rpms
+- rhel-8-appstream-rpms
+for_payload: true
```

Comment:
Is this a CVO managed image? I saw the ticket says it is not a CVO managed image. Please change this value to `false` if it is not.